### ISSUE TITLE
Support for undefined values in css in js objects

### ISSUE THREAD

**Issue by mickeyreiss-visor** [postcss/autoprefixer#906]
I'm seeing a confusing error when I process css in JS with an undefined value. 

This is a real use-case. I'm using postcss + autoprefixer in a react.js. It seems that undefined values are supported by React's inline style attribute.

Steps to reproduce in node console:

```js
const postcssJs = require('postcss-js')
const autoprefix = postcssJs.sync([autoprefixer])
autoprefix({ height: undefined })
/* → TypeError: Cannot read property 'indexOf' of undefined
    at OldValue.check (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/autoprefixer/lib/old-value.js:23:18)
    at /Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/autoprefixer/lib/processor.js:266:30
    at /Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/postcss-js/node_modules/postcss/lib/container.js:188:28
    at /Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/postcss-js/node_modules/postcss/lib/container.js:144:26
    at Root.each (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/postcss-js/node_modules/postcss/lib/container.js:110:22)
    at Root.walk (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/postcss-js/node_modules/postcss/lib/container.js:143:21)
    at Root.walkDecls (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/postcss-js/node_modules/postcss/lib/container.js:186:25)
    at Processor.remove (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/autoprefixer/lib/processor.js:221:20)
    at plugin (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/autoprefixer/lib/autoprefixer.js:84:32)
    at LazyResult.run (/Users/mickeyreiss/code/github.com/aavnsh/Glue/clientApp/node_modules/postcss-js/node_modules/postcss/lib/lazy-result.js:270:20)
*/
```

Note, `null` works as expected: 
```js
autoprefix({ height: null }) // → {}
```

I would have expected postcss/autoprefixer to handle `undefined` and `null` similarly. If this is a breaking change that is not possible for some reason, I'd expect a more direct error message that validates the input and explains the constraint.

Thanks!

**Comment by ai** [comment]
We can ignore all `undefined` declarations in `postcss-js`. So `autoprefix({ height: undefined })` → `{ }`. Is it correct behaviour?

**Comment by mickeyreiss-visor** [comment]
@ai That behavior would make sense to me 😃

**Comment by ai** [comment]
Done https://github.com/postcss/postcss-js/commit/99765c655155082278a81c3752071ca7b53657f7

**Comment by ai** [comment]
Released in `postcss-js` 1.0.1

**Comment by mickeyreiss-visor** [comment]
Awesome. Thanks, @ai!
